bpftrace/Dockerfile

```diff
@@ -0,0 +1,118 @@
+# ==================== bpftrace-builder image ====================
+FROM ghcr.io/cybozu/ubuntu-dev:24.04 AS builder
+
+# https://github.com/bpftrace/bpftrace/blob/master/docker/Dockerfile.ubuntu
+# Added: libcurl4-openssl-dev and libedit-dev
+#   - possibly because of the difference in base image
+# Removed: libbpf-dev libbpfcc-dev
+#   - to use locally-built bcc and libbpf
+RUN apt-get update && apt-get install -y \
+    asciidoctor \
+    binutils-dev \
+    bison \
+    build-essential \
+    clang \
+    cmake \
+    flex \
+    libcereal-dev \
+    libelf-dev \
+    libiberty-dev \
+    libpcap-dev \
+    llvm-dev \
+    liblldb-dev \
+    libclang-dev \
+    systemtap-sdt-dev \
+    zlib1g-dev \
+    libcurl4-openssl-dev \
+    libedit-dev
+
+WORKDIR /build
+COPY src/bcc ./bcc
+COPY src/bpftrace ./bpftrace
+COPY src/libbpf ./libbpf
+
+ENV CC=/usr/bin/clang
+ENV CXX=/usr/bin/clang++
+
+# -------------------- Build bcc --------------------
+# https://github.com/iovisor/bcc/blob/master/INSTALL.md#ubuntu---source
+#
+# bcc and bpftrace are closely-related tools and bpftrace depends on bcc.
+# For their difference, see:
+# - Linux Extended BPF (eBPF) Tracing Tools
+#   https://www.brendangregg.com/ebpf.html
+WORKDIR /build/bcc
+
+# Removed: arping, netperf, and iperf
+#   - they are installed in upstream images to run examples
+# Please see the upstream instruction for further info.
+RUN apt-get install -y --no-install-recommends \
+        zip \
+        bison \
+        build-essential \
+        cmake \
+        flex \
+        git \
+        libedit-dev \
+        libllvm18 \
+        llvm-18-dev \
+        libclang-18-dev \
+        python3 \
+        zlib1g-dev \
+        libelf-dev \
+        libfl-dev \
+        python3-setuptools \
+        liblzma-dev \
+        libdebuginfod-dev \
+        libpolly-18-dev
+
+# Missing -DREVISION causes an error, although it is not documented
+# ref. https://github.com/iovisor/bcc/blob/master/cmake/version.cmake
+RUN mkdir build && \
+    cmake -B build -DREVISION=0.33.0 && \
```

Comment:
`REVISION` is hard coded here.
I suspect that this is bcc version, right?
If so, it would be better to use ARG or to document somewhere.

Reply:
Sorry I missed it!